ci/action.yml

```
@@ -0,0 +1,149 @@
+name: "Mozilla Terraform CI"
+description: "Terraform CI for Mozilla SRE Teams"
+
+inputs:
+  readme_check:
+    description: "Check Terraform path for README.md"
+    required: false
+    default: "false"
+  tfpath:
+    description: "Terraform git repository path"
+    required: false
+    default: "."
+
+runs:
+  using: "composite"
+  steps:
+    - name: Clone Repository
+      uses: actions/checkout@v2
+    - name: Install jq
+      shell: bash
+      run: |
+        echo "****** INSTALL JQ ******"
+
+        if ! command -v "jq" > /dev/null; then
+          sudo apt-get update
+          sudo apt-get install jq
+        fi
+    - name: Install json2hcl
+      shell: bash
+      run: |
+        echo "****** INSTALL JSON2HCL ******"
+
+        if ! command -v "json2hcl" > /dev/null; then
+          curl -SsL https://github.com/kvz/json2hcl/releases/download/v0.0.6/json2hcl_v0.0.6_linux_amd64 | sudo tee /usr/local/bin/json2hcl > /dev/null
+          sudo chmod 755 /usr/local/bin/json2hcl
+          json2hcl -version
+        fi
+    - name: Install TFEnv
+      shell: bash
+      run: |
+        echo "****** INSTALL TFENV ******"
+
+        if ! command -v "tfenv" > /dev/null; then
+          git clone https://github.com/tfutils/tfenv.git ~/.tfenv
+          sudo rm /usr/local/bin/tfenv || echo "INFO: No tfenv installed"
+          sudo rm /usr/local/bin/terraform || echo "INFO: No terraform installed"
+          sudo ln -s ~/.tfenv/bin/* /usr/local/bin > /dev/null
+        fi
+    - name: Terraform Install
+      working-directory: ${{ inputs.tfpath }}
+      shell: bash
+      run: |
+        echo "****** INSTALL TERRAFORM ******"
+
+        if [ -f .terraform-version ]; then
+          tfenv install "$(cat .terraform-version)" > /dev/null
+        else
+          tfenv install 1.0.0 && tfenv use 1.0.0 > /dev/null
+        fi
+    - name: Check README.md
+      if: inputs.readme_check == 'true'
+      shell: bash
+      working-directory: ${{ inputs.tfpath }}
+      run: |
+        echo "****** CHECK README.MD ******"
+
+        if [[ ! -f .terraform-version ]]; then
```

```suggestion
        if [[ ! -f README.md ]]; then
```

goddamnit i found my own error